tools/build_file_generator/third_party_deps/BUILD

```diff
@@ -21,42 +21,41 @@ checkstyle_test(
     target = ":third_party_deps_analyzer",
 )
 
-# '%START%' and '%END%' are markers for 'third_party_deps_updater.sh' script.
-# Don't delete it while 'third_party_deps_updater.sh' script is used.
+# '%START%' and '%END%' are markers for 'third_party_deps_updater.sh' script and ThirdPartyDepsAnalyzer class.
+# Don't delete it while 'third_party_deps_updater.sh' script and ThirdPartyDepsAnalyzer class are used.
 THIRD_PARTY_DEPS_GROUP = [
     # %START%
```

Comment:
Maybe we should rename to `THIRD_PARTY_DEPS_START`, and similarly for `END`?

Reply:
Done.